src/endpoint/iam/ops/iam_list_user_tags.js

```diff
@@ -14,20 +14,19 @@ async function list_user_tags(req, res) {
     const params = {
         username: req.body.user_name,
         marker: req.body.marker,
-        max_items: iam_utils.parse_max_items(req.body.max_items) ?? iam_constants.DEFAULT_MAX_ITEMS,
+        max_items: iam_utils.parse_max_items(req.body.max_items) ?? iam_constants.DEFAULT_MAX_TAGS,
     };
 
-    dbg.log1('To check that we have the user we will run the IAM GET USER', params);
-    iam_utils.validate_params(iam_constants.IAM_ACTIONS.GET_USER, params);
-    await req.account_sdk.get_user(params);
-
-    dbg.log1('IAM LIST USER TAGS (returns empty list on every request)', params);
+    dbg.log1('IAM LIST USER TAGS', params);
+    iam_utils.validate_params(iam_constants.IAM_ACTIONS.LIST_USER_TAGS, params);
+    const reply = await req.account_sdk.list_user_tags(params);
+    dbg.log2('list_user_tags reply', reply);
 
     return {
         ListUserTagsResponse: {
             ListUserTagsResult: {
-                Tags: [],
-                IsTruncated: false,
+                Tags: reply.tags || [],
+                IsTruncated: reply.is_truncated || false
```

Comment:
Please don't default the value here and only return what you had in the reply.
```suggestion
                Tags: reply.tags,
                IsTruncated: reply.is_truncated
```

For the NC default value, you can do something similar to this:
https://github.com/noobaa/noobaa-core/blob/a566cda9239262765570ed38a35fb6a218cc3576/src/sdk/accountspace_fs.js#L550-L559